server/src/main/java/com/groep9/moviemingleserver/data/MovieDAO.java

```diff
@@ -110,8 +110,31 @@ public static boolean fetchLatestMovies() {
 
     @Override
     public boolean save(Movie item) {
-        // TODO(jonas) do an upsert here instead of a regular insert.
-        return false;
+
```

Comment:
decided against an update or insert, just checking if the value already exists, then inserting.